orchagent/portsorch.cpp

```diff
@@ -1179,15 +1251,25 @@ bool PortsOrch::addBridgePort(Port &port)
         return false;
     }
 
+    m_portList[port.m_alias] = port;
     SWSS_LOG_NOTICE("Add bridge port %s to default 1Q bridge", port.m_alias.c_str());
 
     return true;
 }
 
-bool PortsOrch::removeBridgePort(Port port)
+bool PortsOrch::removeBridgePort(Port &port)
 {
     SWSS_LOG_ENTER();
 
+    if (port.m_vlan_members.size() > 0)
+    {
+        for (auto &vme: port.m_vlan_members)
+        {
+            SWSS_LOG_DEBUG("Port %s still in VLAN %d", port.m_alias.c_str(), vme.first);
```

Comment:
SWSS_LOG_ERROR, this should not happen at orchagent level, the dependency resolution should be handled by the mgr level.

Reply:
As previous explanation.    
We may argue there could be different way of implementation if it is too confusing.

Reply:
the return value of the function needs to be consistent, true mean removed, false means not removed.

Reply:
Ok,  it is a little confusing.  Will do the m_vlan_members set empty check outside of this function.